wordpress/Caddyfile

```diff
@@ -1,2 +1,10 @@
 localhost:8080
-fastcgi / unix:/var/run/php5-fpm.sock php
+root ./html/
+gzip
+log ./access.log
+errors ./error.log
```

Comment:
We also don't need to enable access and error logging by default (particularly logs that don't roll when they get large -- although some sort of rotation will probably become the default setting in a future version of Caddy).